### PR TITLE
Changed certificate renewal process to use hooks

### PR DESCRIPTION
**Pull request by groodle**
certbot-auto was removed and certbot installed according to the official instructions. 

The old renewal process of creating a custom file in /etc/cron.weekly/ was removed. This is because when certbot is installed a systemd timer is created to handle certificate renewals. 

Nginx and Apache are reloaded via  pair of hook files.

### CODE REVIEW

File: resources/install-letsencrypt-cert.sh [hunk 1]
```diff
@@ -23,25 +23,37 @@ echo "by providing an email address for important account notifications"
...
-fi
-
-CRON_FILE="/etc/cron.weekly/letsencrypt-renew"
-if [ ! -d "/etc/cron.weekly" ] ; then
-    mkdir "/etc/cron.weekly"
+if [ ! -x "$(command -v certbot)" ] ; then
+    DISTRO=$(lsb_release -is)
+    DISTRO_VERSION=$(lsb_release -rs)
+    if [ "$DISTRO" = "Debian" ]; then
+        apt-get update
```
Comment (xChat) [inline resources/install-letsencrypt-cert.sh:30]: askdflwjeo

File: resources/install-letsencrypt-cert.sh [hunk 2]
```diff
@@ -23,25 +23,37 @@ echo "by providing an email address for important account notifications"
...
+        apt-get -y install certbot
+    elif [ "$DISTRO" = "Ubuntu" ]; then
+        if [ "$DISTRO_VERSION" = "20.04" ] || [ "$DISTRO_VERSION" = "19.10" ]; then
+                apt-get update
+                apt-get -y install software-properties-common
+                add-apt-repository -y universe
```
Comment (xChat) [inline resources/install-letsencrypt-cert.sh:36]: owieorwe**asdf**

File: resources/install-letsencrypt-cert.sh [hunk 3]
```diff
@@ -23,25 +23,37 @@ echo "by providing an email address for important account notifications"
...
-CRON_FILE="/etc/cron.weekly/letsencrypt-renew"
-if [ ! -d "/etc/cron.weekly" ] ; then
-    mkdir "/etc/cron.weekly"
+if [ ! -x "$(command -v certbot)" ] ; then
+    DISTRO=$(lsb_release -is)
+    DISTRO_VERSION=$(lsb_release -rs)
+    if [ "$DISTRO" = "Debian" ]; then
+        apt-get update
+        apt-get -y install certbot
+    elif [ "$DISTRO" = "Ubuntu" ]; then
```
Comment (saghul) [inline resources/install-letsencrypt-cert.sh:32]: Maybe we should error here, if it's none of these?

File: resources/install-letsencrypt-cert.sh [hunk 4]
```diff
@@ -51,13 +63,17 @@ if [ -f /etc/nginx/sites-enabled/$DOMAIN.conf ] ; then
         chmod u+x $TURN_HOOK
         sed -i "s/jitsi-meet.example.com/$DOMAIN/g" $TURN_HOOK
 
-        ./certbot-auto certonly --noninteractive \
+        cp /usr/share/jitsi-meet-certbot-config/0001-nginx-certbot-deploy.sh $NGINX_HOOK
```
Comment (saghul) [inline resources/install-letsencrypt-cert.sh:66]: where does this hook file come from?

**Comment by saghul** [comment]
Superseded by https://github.com/jitsi/jitsi-meet/pull/6986

**Pull request closed without merging**